mlrun/datastore/base.py

```diff
@@ -262,6 +485,10 @@ def as_df(
         file_url = self._sanitize_url(url)
         is_csv, is_json, drop_time_column = False, False, False
         file_system = self.filesystem
+        create_partition_path = kwargs.pop("create_partition_path", True)
+        time_partitioning_granularity = kwargs.pop(
```

Comment:
These new APIs seem to be undocumented? I don't think the solution should force users to tweak these (even if only in some cases).